.github/workflows/cclf-import-apply.yml

```diff
@@ -0,0 +1,44 @@
+name: cclf-import apply terraform
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/cclf-import-apply.yml
+      - terraform/modules/bucket/**
+      - terraform/modules/key/**
+      - terraform/modules/function/**
+      - terraform/modules/queue/**
+      - terraform/modules/subnets/**
+      - terraform/modules/vpc/**
+      - terraform/services/cclf-import/**
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  terraform-apply:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./terraform/services/cclf-import
+    strategy:
+      fail-fast: false
+      matrix:
+        app: [bcda]
+        env: [dev, test, prod]
```

Comment:
Hmm we'll need to import CCLF files in opensbx as well, but I realized something while adding values to parameter store. Will follow up in slack 🤔

Reply:
Actually, maybe I'm overthinking this. Opensbx probably doesn't need this lambda at this point in time 😄

Reply:
Sounds good, plan looks like it's running correctly!